book/website/collaboration/academic-industry/intro-case-studies-and-best-practices-in-academic-industry-collaboration.md

```diff
@@ -0,0 +1,22 @@
+(cl-academic-industry-best-practices)=
+# Introduction
+The creation of this subchapter was motivated by a need to address the realities of navigating the complex relationship between Open Science and Industry, particularly in cases where research is partly or fully funded by industry. A balance must be struck between producing research that meets the priorities of the sponsoring organisation and producing research that is open, shareable, and beneficial to the community. 
+
+Researchers and practitioners championing Open Science could benefit from case studies and lessons learnt from others who have successfully and/or unsuccessfully navigated this aspect of Academic-Industry collaborations.
+
+This subchapter is a continuously growing resource library of case studies in academic-industry collaboration, as well as a collection of existing resources on academic-industry best practices.
+
```

Comment:
you need to add the heading thing here and below for each e.g 
(cl-academic-industry-best-practices-who)=

Reply:
added now!